hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestOzoneFileSystem.java

```diff
@@ -248,6 +253,34 @@ private void testDeleteCreatesFakeParentDir() throws Exception {
     assertEquals(parentKey, parentKeyInfo.getName());
   }
 
```

Comment:
It would be great if we add a failure case here like there is an unknown key in a list of known keys. We would like to also test exceptions.